packages/core/src/components/Button/Button.jsx

```diff
@@ -87,7 +88,11 @@ export class Button extends React.PureComponent {
       delete attrs.type;
     }
 
-    return <ComponentType {...attrs}>{this.props.children}</ComponentType>;
+    return (
+      <ComponentType ref={this.props.buttonRef} {...attrs}>
```

Comment:
since you already defined `buttonRef` above, just for sake of using it, can you remove the `this.props` here?